htdocs/comm/propal/index.php

```diff
@@ -139,7 +140,7 @@
 	print '</tr>';
 
 	foreach ($listofstatus as $status) {
-		$dataseries[] = array($propalstatic->LibStatut($status, 1), (isset($vals[$status]) ? (int) $vals[$status] : 0));
+		$dataseries[] = array(html_entity_decode($propalstatic->LibStatut($status, 1)), (isset($vals[$status]) ? (int) $vals[$status] : 0));
```

Comment:
I would prefer another way to fix: Not having entity encoded the value returned by LibStatus. Because this value can be stored in memory, used by other output than HTML, etc...